src/components/NoteInput/NoteInput.css

```diff
@@ -1,8 +1,21 @@
 .note-form {
-  max-width: 25rem;
-  margin: 1rem auto;
+  width: 25rem;
+  margin: 1rem;
   border-radius: var(--border-radius);
-  border: 2px solid var(--primary-color);
+  background-color: var(--white);
+}
+
+.form-open {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
```

Comment:
you can do top 0 left 0 right 0 bottom 0 instead of height and with as 100%

Reply:
Okay, thanks.